### PR TITLE
feature: added chat and chat row components

### PR DESCRIPTION
**Pull request by elpollo12**
### Description

Added chat and chat row components

fixes # (issue)

#31 

#### Screenshots

scrnli_25_10_2021 20-16-54

### CODE REVIEW

File: src/pages/Chat.jsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { useParams, useHistory } from 'react-router-dom'
 import deliveryManWhite from '../assets/delivery-chat-white.png'
 /* import assistant from '../assets/asistant-chat.png'
-import sendButton from '../assets/send-chat.png'
-import userIcon from '../assets/user-icon.png' */
+import sendButton from '../assets/send-chat.png' */
```
Comment (Israel-Laguan) [inline src/pages/Chat.jsx]: remove comments

File: src/components/Chat/MessageBox.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  background: #e1e1e1;
+  border-radius: 15px;
+  color: #646464;
+  font-size: 1em;
+  min-height: 52px;
+  margin: 0 1vw;
+  line-height: 30px;
+  width: 80vw;
+  @media (min-width: 960px) {
+    width: 40vw;
```
Comment (Israel-Laguan) [inline src/components/Chat/MessageBox.jsx]: please add max-width to avoid issues on widescreen

File: src/components/Chat/MessageBox.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components'
+
+const MessageBox = styled.div`
+  display: inline-block;
+  padding: 10px;
+  background: #e1e1e1;
+  border-radius: 15px;
+  color: #646464;
+  font-size: 1em;
```
Comment (Israel-Laguan) [inline src/components/Chat/MessageBox.jsx]: Consider using rem as parent maybe doesn't have font size defined

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]
Also please move all the files into one named Chat.styled.jsx as them are small

**Comment by elpollo12** [comment]
The code was updated @Israel-Laguan

**Review by Israel-Laguan — APPROVED** [review]

**Comment by denisosuna** [comment]
Looks good, I think is ready to go, Good Job @elpollo12

**Pull request merged**